doc/rst/developer/bestPractices/StandardModuleStyle.rst

```diff
@@ -114,7 +114,23 @@ could have:
  * ``createStringWithBorrowedBuffer`` or
  * ``string.createWithBorrowedBuffer``.
 
+Accessors
++++++++++
+
+Accessor methods are methods intended to return a distinct piece of information
```

Comment:
I think I've heard the definition that accessor methods return quasi-static pieces of information, or pieces of information that could reasonably be implemented as fields (regardless of whether they actually are).

Rewording suggestion: "We define accessor methods as those that return a distinct and relatively-static piece of information which could be reasonably implemented as a field".

Reply:
I don't think the static-ness of the information is important for whether it is considered an accessor, so I'd like to leave that portion out.  How does the following reword sound?

```
Accessor methods are methods intended to return a distinct piece of information
about the type on which they are defined.  This piece of information could be
reasonably implemented as a field.  Typically, such methods are defined without
parentheses, allowing them to appear similar to [...]
```

Reply:
Yeah that sounds good!